test-integration/helpers/describe.coffee

```diff
@@ -49,12 +51,31 @@ describe = (name, cb) ->
       # Wait 20sec for the browser to start up
       @timeout(20 * 1000, true)
 
+      chromeOptions = new chrome.Options()
+      prefs = {
+        download:
+          "default_directory": "test-integration/downloaded-files"
+      }
+      chromeOptions.setUserPreferences(prefs)
+
+      firefoxProfile = new firefox.Profile()
+      firefoxProfile.setPreference("browser.helperApps.neverAsk.saveToDisk", "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")
+      firefoxProfile.setPreference("browser.helperApps.neverAsk.openFile", "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")
+      firefoxProfile.setPreference("browser.helperApps.alwaysAsk.force", false)
+      firefoxProfile.setPreference("browser.download.useDownloadDir", true)
+      firefoxProfile.setPreference("browser.download.manager.showWhenStarting",false)
+      firefoxProfile.setPreference("browser.download.folderList", 2)
+      firefoxProfile.setPreference("browser.download.dir", "#{__dirname}/../downloaded-files")
```

Comment:
good idea.  they were separate b/c `/../` didn't resolve in chrome; to workaround i changed it to: `@downloadDirectory = "#{__dirname}/".replace('helpers/', 'downloaded-files')`